website/docs/r/compute_instance_server.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_compute_instance_server"
+sidebar_current: "docs-scaleway-resource-compute-instance-server"
+description: |-
+  Manages Scaleway Compute Instance servers.
+---
+
+# scaleway_compute_instance_server
+
+Creates and manages Scaleway Compute Instance servers. For more information, see [the documentation](https://developers.scaleway.com/en/products/instance/api/#servers-8bf7d7).
+
+## Example Usage
+
+```hcl
+resource "scaleway_compute_instance_server" "web" {
+  type = "DEV1-S"
+  image_id = "f974feac-abae-4365-b988-8ec7d1cec10d"
+}
+```
+
```

Comment:
Add more complex example interacting with other resource:
```hcl

resource "scaleway_compute_instance_ip" "myip" {
  server_id = "${scaleway_compute_instance_server.webserver.id}"
}

resource "scaleway_compute_instance_volume" "data" {
  size_in_gb = 100
}

resource "scaleway_compute_instance_server" "webserver" {
  image_id = "f974feac-abae-4365-b988-8ec7d1cec10d"
  type  = "DEV1-S"
  additional_volume_ids = [ "${scaleway_compute_instance_volume.data.id}" ]

  provisioner "remote-exec" {
    inline = [
      "touch /tmp/fakeconfig",
    ]
  }
}

```

Reply:
Yes, check the MR description. :)